test/test_jit.py

```diff
@@ -2052,6 +2052,22 @@ def forward(self, x, weight, bias):
         res = m(x, weight, bias)
         self.assertEqual(res, ref_res)
 
+    def test_finalize(self):
```

Comment:
Please move test to #33927

Reply:
this test is only possible after this PR, but I can add another test there that tests conv2d